File: SDKs/yodlee.com/1.1.0/python/src/sdk/models/shared/cobrandloginresponse.py
import dataclasses
from typing import Optional
from dataclasses_json import dataclass_json
from sdk import utils
from ..shared import cobrandsession as shared_cobrandsession


@dataclass_json
@dataclasses.dataclass
class CobrandLoginResponse:
    application_id: Optional[str] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('applicationId') }})
    cobrand_id: Optional[int] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('cobrandId') }})
    locale: Optional[str] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('locale') }})
    session: Optional[shared_cobrandsession.CobrandSession] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('session') }})
    

File: SDKs/yodlee.com/1.1.0/typescript/src/sdk/models/shared/cobrandloginresponse.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";
import { CobrandSession } from "./cobrandsession";



export class CobrandLoginResponse extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=applicationId" })
  applicationId?: string;

  @SpeakeasyMetadata({ data: "json, name=cobrandId" })
  cobrandId?: number;

  @SpeakeasyMetadata({ data: "json, name=locale" })
  locale?: string;

  @SpeakeasyMetadata({ data: "json, name=session" })
  session?: CobrandSession;
}
